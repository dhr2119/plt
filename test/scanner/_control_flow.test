if elsif else while for return main break
