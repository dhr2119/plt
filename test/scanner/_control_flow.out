IF
ELSIF
ELSE
WHILE
FOR
RETURN
MAIN
BREAK
